integration/integration_test.go

```diff
@@ -208,3 +208,10 @@ func TestHttpIngress(t *testing.T) {
 		}),
 	)
 }
+
+func TestRuntimeReflection(t *testing.T) {
```

Comment:
@matt2e a similar pattern to this is how you'd test the `ftltest` changes I think